# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1323017307 0
#      Sun Dec 04 16:48:27 2011 +0000
# Node ID b4d399c975de959bd00720a8a5f429c66b483262
# Parent  7ab497513c1a96cc9941f6f10764a4a708b306fc
Export new added functions/classes (Woe32)

* liboctave/oct-locbuf.h (octave_chunk_buffer::clear): Add OCTAVE_API.
* liboctave/singleton-cleanup.h (class singleton_cleanup): Likewise.

diff --git a/liboctave/oct-locbuf.h b/liboctave/oct-locbuf.h
old mode 100644
new mode 100755
--- a/liboctave/oct-locbuf.h
+++ b/liboctave/oct-locbuf.h
@@ -69,17 +69,17 @@ class octave_chunk_buffer
 public:
 
   OCTAVE_API octave_chunk_buffer (size_t size);
 
   OCTAVE_API virtual ~octave_chunk_buffer (void);
 
   char *data (void) const { return dat; }
 
-  static void clear (void);
+  static OCTAVE_API void clear (void);
 
 private:
 
   // The number of bytes we allocate for each large chunk of memory we
   // manage.
   static const size_t chunk_size;
 
   // Pointer to the end end of the last allocation.
diff --git a/liboctave/singleton-cleanup.h b/liboctave/singleton-cleanup.h
old mode 100644
new mode 100755
--- a/liboctave/singleton-cleanup.h
+++ b/liboctave/singleton-cleanup.h
@@ -1,14 +1,15 @@
 #if !defined (octave_singleton_cleanup_h)
 #define octave_singleton_cleanup_h 1
 
 #include <set>
 
 class
+OCTAVE_API
 singleton_cleanup_list
 {
 protected:
 
   singleton_cleanup_list (void) : fcn_list () { }
 
 public:
 
